x/stakeibc/client/rest/tx.go

```diff
@@ -0,0 +1,21 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	govrest "github.com/cosmos/cosmos-sdk/x/gov/client/rest"
+)
+
+// TODO: Do we need to implement these?
```

Comment:
I'm assuming the answer is no?

Reply:
I'm in the camp of keeping it out unless there's an obvious reason why we need this. I saw other repo's did not implement it beyond this